7주차/[Lv.2]_가장_큰_정사각형_찾기/안혜수.js

```diff
@@ -0,0 +1,14 @@
+function solution(board) {
+  let Max = 0;
+  const [row, col] = [board.length, board[0].length];
+
+  for (let i = 0; i < row; i++) {
+    for (let j = 0; j < col; j++) {
+      if (i > 0 && j > 0 && board[i][j] && board[i - 1][j - 1] && board[i][j - 1] && board[i - 1][j]) {
```

Comment:
```suggestion
      if (i > 0 && j > 0 && board[i][j]) {
```

이미 `i > 0 && j > 0`에서 `i`와 `j`의 값이 기존 범위를 넘어가는 것을 방지했기 때문에 뒤의 조건은 생략 가능하다고 생각합니다!! 🤔